kotlin/android/app/src/main/java/dev/firezone/android/core/di/DataModule.kt

```diff
@@ -21,6 +22,7 @@ class DataModule {
         @ApplicationContext context: Context,
     ): Bundle = (context.getSystemService(Context.RESTRICTIONS_SERVICE) as android.content.RestrictionsManager).applicationRestrictions
 
+    @Singleton
```

Comment:
Without this line, Hilt was creating 2 `Repository` instances, one for each ViewModel. I have no clue how Hilt works internally so I'm not sure why it did that.